### PR TITLE
Configurable driver memory

### PR DESCRIPTION
**Pull request by acidghost**
@velvia Here it is :smiley:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,4 +11,6 @@ test-reports/
 # ignore deployment configs
 config/*.conf
 config/*.sh
+job-server/config/*.conf
```
Comment (velvia) [inline .gitignore]: Nice!  Should have had that ages ago.

File: job-server/config/local.sh.template [hunk 2]
```diff
@@ -10,6 +10,8 @@ APP_GROUP=spark
 INSTALL_DIR=/home/spark/job-server
 LOG_DIR=/var/log/job-server
 PIDFILE=spark-jobserver.pid
+DRIVER_MEMORY=1G
+SPARK_VERSION=1.3.1
```
Comment (velvia) [inline job-server/config/local.sh.template]: Where is SPARK_VERSION used again?

Comment (acidghost) [inline job-server/config/local.sh.template]: Here https://github.com/spark-jobserver/spark-jobserver/blob/master/project/Dependencies.scala#L30

Comment (velvia) [inline job-server/config/local.sh.template]: @acidghost ah right.  Thanks!  Going to merge.

**Pull request merged**